tests/foreman/ui/test_host.py

```diff
@@ -1913,3 +1925,90 @@ def test_positive_gce_provision_end_to_end(
         finally:
             gce_client.disconnect()
             skip_yum_update_during_provisioning(template='Kickstart default finish', reverse=True)
+
+
+@tier4
+@skip_if_not_set('gce')
+def test_positive_gce_cloudinit_provision_end_to_end(
+        session,
+        module_org,
+        module_loc,
+        module_os,
+        gce_domain,
+        gce_hostgroup,
+        gce_client
+):
+    """Provision Host on GCE compute resource
+
+    :id: 6ee63ec6-2e8e-4ed6-ae48-e68b078233c6
+
+    :expectedresults: Host is provisioned successfully
+
+    :CaseLevel: System
+    """
+    name = gen_string('alpha').lower()
+    hostname = u'{0}.{1}'.format(name, gce_domain.name)
+    gceapi_vmname = hostname.replace('.', '-')
+    root_pwd = gen_string('alpha', 15)
```

Comment:
There is no issue to keep limit `15 `,
one thought on it, can we keep the password length to `8` because that is a minimum limit ... and would help to see if with this limit GCE able to provision the VM with that minimum limit ? what do you think ?

Reply:
Ofcourse, Thanks for the thought ! Updated !